The install instruction is not available in doc repo nor translatable

DelazJ opened the pull request:
Actually the install doc is not available through qgis.org nor translated
Remove SPIT plugin mention
Also point readers to instructions to build versions other than master (fix https://github.com/qgis/QGIS-Website/issues/382)

DelazJ commented on doc/INSTALL.t2t:
```diff
@@ -85,22 +85,19 @@ liable to any duties or guarantees. The editors and publishers do not take
-You can download this document as part of the QGIS 'User and 
-Installation Guide' in HTML and PDF format via http://qgis.org. A current 
-version is also available at:
+The current instructions apply to the **master** branch of QGIS. If you wish
```
@jef-n is there a way to have **master** replaced automatically by the current branch name (so that comment stays coherent with the releases)?

DelazJ commented:
Failed for unrelated reason

DelazJ commented:
@m-kuhn any idea why i still get _The job exceeded the maximum time limit for jobs, and has been terminated_, please? Thanks

m-kuhn commented:
yes, the cache is for an older version of master. I deleted the cache for this pull request and restarted the build which should build on the current master cache. Unfortunately I don't have an automatic solution to this kind of problems..

DelazJ commented:
Thanks Matthias

DelazJ commented:
Any issue with this PR?

m-kuhn commented:
Sorry @DelazJ , that slipped through the cracks.
Can you rebase to get it into a mergable state (and ping me if I don't realize)

DelazJ committed:
Update the install instructions

- The install doc is not available in documentation nor translated
- SPIT plugin no longer exists
- Mention versioning of instructions depending on branches

DelazJ commented:
@m-kuhn may I?

m-kuhn commented:
Sure, thanks!
